.github/workflows/deploy.yml

```diff
@@ -1,19 +1,40 @@
-name: Deploy docs from master commit
+name: Deploy docs to gh-pages
 on:
   push:
     branches:
       - master
+      - v8_maintenance
+      - v9_maintenance
+  workflow_dispatch:
 jobs:
   deploy-docs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - run: npm ci && npm run bootstrap
+      # Conditional build depending on branch
+      - name: Set build directory based on branch
+        id: set-build-dir
+        run: |
+          if [ "${{ github.ref }}" == "refs/heads/master" ]; then
+            echo "::set-output name=build_dir::./packages/__docs__/__build__"
```

Comment:
could this be set before the if/elif statement? it's the same for every condition